backend/routers/spiritual.py

```
@@ -1093,4 +1095,151 @@ async def generate_birth_chart_for_user(request: Request):
+    message: str = ""
+
+@router.post("/personalized-remedies", response_model=PersonalizedRemediesResponse)
+async def get_personalized_remedies(
+    request: PersonalizedRemediesRequest
+):
+    """
+    Generate personalized spiritual remedies based on birth details and current issues
+    """
+    try:
+        # Extract birth details
+        birth_details = request.birth_details
```

Ruff (0.12.2)

Local variable `birth_details` is assigned to but never used

Remove assignment to unused variable `birth_details`

(F841)